ani-cli

```diff
@@ -278,12 +278,23 @@ anime_selection () {
 episode_selection () {
 	# user input for selecting episodes of a series
 	ep_choice_start="1"
-	if [ "$last_ep_number" -gt 1 ]
+	
+	# using get_dpage_link to get confirmation from episode 0 if it exists,else first_ep_number becomes "1"
+	first_ep_number="0"
+	result=$(get_dpage_link "$anime_id" "$first_ep_number")
+	
+	if [ $result ]; then
+		continue
```

Comment:
Shellcheck didn't like that, otherwise this looks good

Reply:
Explicit -n on line 287 is still missing

Reply:
or 286, somewhere around there

Reply:
that part is stated as a notice and not an error, the error is exit code 1 which i dont know whats causing it because its working just fine